siliconcompiler/schema/schema_cfg.py

```diff
@@ -3557,6 +3440,129 @@ def schema_constraint(cfg, scenario='default', name = 'default'):
             starting with m1 as the lowest routing layer and ending
             with m<n> as the highest routing layer.""")
 
+    # NETS
+    scparam(cfg, ['constraint', 'net', name, 'maxlength'],
+            sctype='float',
+            shorthelp="Constraint: Net max length",
+            switch="-constraint_net_maxlength 'name <float>'",
+            example=[
+                "cli: -constraint_net_maxlength 'nreset 1000'",
+                "api: chip.set('constraint', 'net', 'nreset', 'maxlength', '1000')"],
+            schelp="""
+            Maximum total length of a net.""")
```

Comment:
It's removed from `asic,maxlength` but added back here?. I suppose you could also apply this to {*}, so the help should be updated accordingly.